Update Ledger Polkadot JS Video link (#3049)
w3f/polkadot-wiki: docs/general/ledger.md

## Using on Polkadot-JS Apps

> If you are a beginner, please consider watching this video tutorial on how to connect your Ledger

Reviewer: ```suggestion
Please consider watching this video tutorial on how to connect your Ledger
```

> device with Polkadot JS UI, with your accounts imported to Polkadot browser extension.

Reviewer: ```suggestion
device with the Polkadot-JS UI. Your accounts will need to be imported on the Polkadot-JS browser extension.
```

Reviewer 2: A better way to word it - (as the user can either import it to extension which is the recommended way or directly to the UI as explained in the later section)

```suggestion
device with Polkadot-JS UI, by importing your accounts to Polkadot-JS Extension
```
